app/helpers/application_helper.rb

```diff
@@ -155,69 +155,124 @@ def draw_note_tree_leaves(notes,level,output,key)
     end
   end
 
-  def draw_tree(root, id = nil, concept_schemes = [])
```

Comment:
you removed `draw_tree`, here but it is still used in the file `_treeview`

Reply:
May be the solution would be to just change in _treeview, draw_tree by your tree component